src/solver/type_getters.jl

```diff
@@ -613,6 +602,15 @@ function get_comms_context(parsed_args)
     return comms_ctx
 end
 
+function set_global_logger(comms_ctx)
+    # Set logging to only display for the root process
+    if ClimaComms.iamroot(comms_ctx)
+        Logging.global_logger(Logging.ConsoleLogger(stderr, Logging.Info))
+    else
+        Logging.global_logger(Logging.NullLogger())
+    end
+end
```

Comment:
```suggestion
function silence_non_root_processes(comms_ctx)
    # Set logging to only display for the root process
    if ClimaComms.iamroot(comms_ctx)
        Logging.global_logger(Logging.ConsoleLogger(stderr, Logging.Info))
    else
        Logging.global_logger(Logging.NullLogger())
    end
end
```
Can you also add a dosctring?